### PR TITLE
Add clarification about verifiability

### PR DESCRIPTION
**Pull request by clehner**
Address #751. Adds text from https://github.com/w3c/vc-data-model/issues/751#issuecomment-946074037, incorporating the suggestions by @dlongley and @TallTed.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/829.html" title="Last updated on Dec 6, 2021, 1:46 PM UTC (1e114de)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/829/e77940a...1e114de.html" title="Last updated on Dec 6, 2021, 1:46 PM UTC (1e114de)">Diff</a>

### CODE REVIEW

File: index.html
Comment (TallTed) [inline index.html:325]:
```suggestion
that the truth content of <a>claims</a> encoded in it can be evaluated.
```

File: index.html
Comment (TallTed) [inline index.html:325]: sorry, introduced my own typo.
```suggestion
that the truth content of <a>claims</a> encoded therein can be evaluated.
```

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2021-11-03-vcwg#section4-2) on 2021-11-03

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 4.2. Add clarification about verifiability (pr vc-data-model#829)

_See github pull request [vc-data-model#829](https://github.com/w3c/vc-data-model/pull/829)._




**Brent Zundel:** next PR is 829, clarification about verifiability from charles lehner.  
… don't see charles on the call today, but would encourage folks to review this PR.  

**Kyle Den Hartog:** anytime those SVGs are included, that's a mistake as the SVG changes haven't been merged into main, we may work with ivan to clean this up prior to merge.  

**Brent Zundel:** for other folks creating PRs, since we're merging into the v1.1 branch, rather than basing your changes off of main, base it off of v1.1.  

**Manu Sporny:** once charles fixes the svg, it should be editorial.  

**Brent Zundel:** would anyone object to labeling this as v1.1 editorial?.  

</details>

**Comment by TallTed** [comment]
I do wish GitHub took comments when addressing merge conflicts ... which is what I've just done.  There were two: one was a trailing space; the other was a misplaced `<li>`.

File: terms.html
Comment (msporny) [inline terms.html:161]:
```suggestion
Verification of a credential does not imply evaluation of the truth of the content
of <a>claims</a> encoded in the credential.</a>.
```

**Review by msporny — APPROVED** [review]

**Review by shigeya — APPROVED** [review]

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2021-11-17-vcwg#section3-3) on 2021-11-17

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 3.3. Add clarification about verifiability (pr vc-data-model#829)

_See github pull request [vc-data-model#829](https://github.com/w3c/vc-data-model/pull/829)._




**Brent Zundel:** some review has occurred.  
… charles or TallTed are not here.  
… if anyone has comments please q.  
… the PR is quite straight forward.  

</details>

File: terms.html
Comment (TallTed) [reply terms.html:161]: I can live with @msporny's suggestion, but I think this phrasing is a bit less convoluted --
```suggestion
Verification of a credential does not imply evaluation of the truth
of <a>claims</a> encoded in the credential.</a>.
```

File: index.html
Comment (TallTed) [inline index.html:284]: Matching the later change suggestion --
```suggestion
as defined in this document. Verifiability of a credential does not imply
that the truth of <a>claims</a> encoded therein can be evaluated.
```

File: terms.html
Comment (msporny) [reply terms.html:161]: I like @TallTed's suggestion better.

**Review by brentzundel — APPROVED** [review]
Approved with @TallTed latest changes

File: terms.html
Comment (dlongley) [reply terms.html:161]: +1 for @TallTed's suggestion.

**Review by dlongley — APPROVED** [review]
LGTM w/@TallTed's suggestions.

**Comment by kdenhartog** [comment]
Just noting some of the commits in this PR will make some slight modifications to the git logs even though the content is not changing nor attribution. I think this is fine, but came across this while investigating #837

**Comment by TallTed** [comment]
It occurs to me while re-re-re-re-re-reading this stuff about "verifying a container doesn't imply any verification of the veracity of the contained utterances" that we might also should be more explicit about the part where "*issuing* a container doesn't imply (and certainly doesn't do anything stronger than imply) any verification of the veracity of the contained, not even by the issuer; only that the issuer was willing to irrefutably utter the contained utterances".

It's very tricky, this whole "veracity" thing.  Sneaks up on you when you least expect it.

Thoughts, @msporny @dlongley @brentzundel @burnburn?

**Comment by msporny** [comment]
> It's very tricky, this whole "veracity" thing. Sneaks up on you when you least expect it.

I expect we'll have to do a second pass when we do the v2.0 work. Yes, it's tricky and yes the language in the document needs to be upgraded to make the point you're making @TallTed ... but like the "let's just change the subtitle for the spec, how hard could that be!?" PR... I expect this issue to take just as long. The good news is that I think the group is philosophically aligned... we just need to find the words to express that philosophy in a way that is concise AND nuanced AND correct. Not a small feat.

**Comment by jandrieu** [comment]
@TallTed wrote
> "_issuing_ a container doesn't imply (and certainly doesn't do anything stronger than imply) any verification of the veracity of the contained, not even by the issuer; only that the issuer was willing to irrefutably utter the contained utterances".

Actually, I think it MUST explicitly mean that the issuer is asserting the truth of the claims. IMO, that's what signing means in an assertionMethod verification relationship.

This doesn't change the PR. It's context.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2021-12-01-vcwg#section4-4) on 2021-12-01

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 4.4. Add clarification about verifiability (pr vc-data-model#829)

_See github pull request [vc-data-model#829](https://github.com/w3c/vc-data-model/pull/829)._




**Kyle Den Hartog:** This is a PR to discuss intricacy of what we mean by "verifiable" -- Charles has written a PR, still some time for responses, majority of responses look good -- should be ready to go soon... probably have a few minor suggestions and we're waiting for 14 day merge..  

> *David Chadwick:* which pr is this?.

**Ted Thibodeau Jr.:** That PR will help w/ the subtitle... but we should get rid of the subtitle..  

</details>

File: index.html
Comment (David-Chadwick) [reply index.html:284]: Can we add. "However, the Evidence property can be used by the issuer to help the verifier determine the truth of the <a>claims</a>"

**Comment by David-Chadwick** [comment]
The issuer can help the verifier to determine the truth of the claim, by including the Evidence property in the issued VC. Like @jandrieu I think that the issuer is asserting that to the best of their knowledge the claim is true, by signing it, even if it is not actually true. If the issuer is purposefully asserting that a statement they know to be untrue is a claim, then they could be tried in a court of law for perjury since they have signed the claim. I dont think this is the intention of the VC spec

**Comment by TallTed** [comment]
> [@jandrieu] Actually, I think it MUST explicitly mean that the issuer is asserting the truth of the claims.

I think it can only mean that the issuer is making the claims/statements/assertions in the credential.  One such claim may be their belief that they are making statements of fact.  It may instead be only that they are making statements of opinion, which are not the same as fact, but may nonetheless call for irrefutable attribution.

> [@David-Chadwick] they could be tried in a court of law for perjury since they have signed the claim.

Assertions like this are flat out dangerous for any ecosystem(s) that include(s) VCs.

There is no legal binding of which I'm aware in the standards we've been working on, certainly not criminal, which is where perjury almost always comes in.

Attaching legal meaning and/or consequence, including but not limited to penalties of perjury, to any use of any VC would be up to the legislative body of the relevant region.  Contracts might put VCs to use in various ways, in which case contractual penalty provisions might apply, but these rarely if ever extend to criminal action for perjury, rather relying on civil penalties and the like for contractual enforcement.

**Comment by David-Chadwick** [comment]
@TallTed I am afraid we are already approaching this (eg integrating VCs with eIDAS). The EC has legislation for qualified certificates that says that anything signed with the private key is equal to a hand written signature and can be used in a court of law. Since anyone can be a VC issuer, then one that signs a VC using a qualified certificate has effectively produced the claim in writing and hand signed it, legally speaking.

**Comment by TallTed** [comment]
@David-Chadwick -- I am not in, nor am I a citizen of, the EC, and I am not governed by its legislation. Nor are the vast majority of humans and organizations. "This is true here" does not equate to "this is true everywhere". Appropriate qualifications SHOULD be attached to all statements like "they could be tried in a court of law for perjury since they have signed the claim".

File: terms.html
Comment (clehner) [reply terms.html]: Committed @TallTed's suggestion: ec0793b23443715cfa67738a87386f407bc18d7c

**Comment by clehner** [comment]
Rebased to remove duplicate commits already present on the v1.1 branch. Re: https://github.com/w3c/vc-data-model/pull/829#issuecomment-976607034, https://github.com/w3c/vc-data-model/pull/829#issuecomment-976139322

Old|New|Message|Comments
-|-|-|-
b4aedcf201744f9c24b8a5748602f816229966cb|041a8cdf2e570fa868f8b81f0bad7aef9cd19b1d (already on v1.1 - not from this PR)|Update SVG diagram with SVG fixes. |
e0ec8de9bcd0c809c78c09310ea201d3d3dc023f|41baf902955019441efbdf8ba158446eface280c (already on v1.1 - not from this PR)|Clarify difference between Credentials and Verifiable Credentials. |
1b76151a2cad38b140e15bb2fec2bf17579aa400|830c3ebd14b709eb6f93297bcfe345310129ff7f|Add clarification about verifiability |https://github.com/w3c/vc-data-model/pull/829
9422902fb9fa20b95bd6a845238230c2d029d582, f5ccce3eeee3b56322499ff1197948a98b1a9533|1f299a1947a6b3422054d946909d8b2101475223 (squashed)|Reword sentence about truth content of claims | https://github.com/w3c/vc-data-model/pull/829#discussion_r733211416, https://github.com/w3c/vc-data-model/pull/829#discussion_r733788825
f372e351e6be50414b8eadfd096e6187650df56a|(dropped)|Merge branch 'v1.1' into cel/issue-751 | Thanks @TallTed
ec0793b23443715cfa67738a87386f407bc18d7c|d22247141a031419b4ce4c542ebfb78d420fd576|Simplify reference to truth of claims|https://github.com/w3c/vc-data-model/pull/829#discussion_r751624351

File: index.html
Comment (msporny) [reply index.html:284]: @clehner -- please review @TallTed's requested changes as well as @David-Chadwick's requested ones. This PR is ready to go after you do that.

**Comment by kdenhartog** [comment]
Thanks for taking the time to clean that all up @clehner. This is in line with what I was expecting with the changes related to #837 as well and don't expect any further changes will be needed for alignment with that.

**Comment by kdenhartog** [comment]
I see that the 14 day merge label was applied 14 days ago, but it seems that recent conversations may affect the status of this PR so I'm going to hold off on merging this one for now.

File: index.html
Comment (TallTed) [reply index.html:284]: @David-Chadwick (@msporny @clehner) -- I've incorporated a paraphrase of your addition to my suggested change. Hopefully this still meets with everyone's approval!

Comment (David-Chadwick) [reply index.html:284]: Yes fine thanks.

Comment (clehner) [reply index.html]: Committed the modified suggested change in 4b358f7bb319bc0d8b73cc413a99af44911e8987.

Updated in 1e114de493b03d820c53223baa98554fa4a9650e to add a link to the evidence property definition.

Aside: while searching the Web for the now-removed phrase "truth content", I found it is used in the works of science philosopher Karl Popper; I was not knowingly using the term in that way, but maybe some people will find this interesting.
https://iep.utm.edu/pop-sci/
https://en.wikipedia.org/wiki/Karl_Popper
https://plato.stanford.edu/entries/truthlikeness/

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2021-12-08-vcwg#section2-3) on 2021-12-08

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 2.3. Add clarification about verifiability (pr vc-data-model#829)

_See github pull request [vc-data-model#829](https://github.com/w3c/vc-data-model/pull/829)._




**Brent Zundel:** Clarification on verifiability. This has had some back and forth for a little while. I believe all the requested changes have been addressed at this point. Charles, can you give a current status on this PR?.  

**Charles Lehner:** Sure, I believe I have addressed the changes requested. I added an additional part to link to the evidence section. It's in a normative section, but it's just making it into a term definition, that's all..  

**Brent Zundel:** Any other feedback? Several approvals ... it seems like something we could merge now..  
… Any opposition to merging now? Plenty of time to review and respond to it..  
… Hearing no objections. I'll leave that to you to do, Manu..  
… That's the PRs. We have 13 minutes left so we might actually touch on some issues today..  

</details>

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**
